api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
```

Comment:
I have added a change to support this. please see https://github.com/kaschioudi/pkp-lib/blob/107ca08bd08465d096917a5fda0fa57db8ce2b22/classes/core/APIRouter.inc.php#L84-L92 and https://github.com/kaschioudi/ojs/tree/2163-1503-fix/api/v1/backend/users